utils/api.js

```
@@ -58,4 +58,13 @@ const resetPassword = ({ newPassword, captchaToken, recoverToken }) => {
     }
   );
 };
-export { login, signUp, validateUserToken, recoverPassword, resetPassword };
+
+const acceptTerms = ({ token }) => {
+  return apiClient.post(`/users//accept-latest-legal-terms`, {}, {
```

ESLint

[error] Replace ``/users//accept-latest-legal-terms`,·{},` with `⏎····`/users//accept-latest-legal-terms`,⏎····{},⏎···`

(prettier/prettier)